docs/source/routing/security/request-limits.mdx

```diff
@@ -273,6 +275,17 @@ Before increasing this limit significantly consider testing performance
 in an environment similar to your production, especially if some clients are untrusted.
 Many concurrent large requests could cause the router to run out of memory.
 
+### `experimental_http1_max_request_headers`
+
+Limit the maximum number of headers of incoming HTTP1 requests.
+The default value is 100 headers.
+
+If router receives more headers than the buffer size, it responds to the client with `431 Request Header Fields Too Large`.
+
+### `experimental_http1_max_request_buf_size`
+
+Limit the maximum buffer size for the HTTP1 connection. Default is ~400kb.
+
```

Comment:
Should one of these two files link to the other instead of duplicate content?

Reply:
I followed the lead of `http_max_request_bytes` and it's duplicated in those to places.
So I just keep it sync with this PR, but definitely something that need to be improved.